index.html

```diff
@@ -50,6 +52,7 @@
                 <a href="#events"><li>Events <div class="nav-line"></li></a>
                 <a href="#committe"><li>Committee <div class="nav-line"></li></a>
                 <a href="https://www.bcusu.com/organisation/9907/"><li>Membership <div class="nav-line"></li></a>
+                <a href="https://hacshub.luketainton.co.uk/hacshub"><li>HaCSHub <div class="nav-line"></li></a>
```

Comment:
Imma allow this right now, however in our rethink of the site we need a section/page for member projects to be fair to everyone. Also i need to get you the hacs.tech domain on this